src/ziggurat/transformer.clj

```diff
@@ -0,0 +1,33 @@
+(ns ziggurat.transformer
```

Comment:
Since this is a timestamp transformer, we should name it as such. There can be other transformers in ziggurat as well.

Reply:
Namespace is changed